kiwi/storage/subformat/ova.py

```diff
@@ -85,10 +85,18 @@ def create_image_format(self) -> None:
         except OSError:
             pass
         ovftool_options = []
-        if CommandCapabilities.has_option_in_help(
-            ovftool, '--shaAlgorithm', raise_on_error=False
-        ):
-            ovftool_options.append('--shaAlgorithm=SHA1')
+        possible_options = [
+            '--shaAlgorithm',
+            '--allowExtraConfig',
+            '--exportFlags',
+        ]
+        for option in possible_options:
+            if CommandCapabilities.has_option_in_help(
```

Comment:
Not sure do we really need to check ovftool are these options are in help output as they are in the ovftool at least for last 6 years.

Reply:
yeah it's a bit overkill and I might be overly cautious but the checks mainly exists because ovftool is a proprietary implementation and for those I have little trust with regards to compatibility. If you feel confident about it, or if you are connected in the development of ovftool I'm ok to remove the capabilities check.

Reply:
I see.
It makes sense.
I'm not connected to ovftool development, so can't be 100% sure that they won't change anything in future...